index.html

```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Donald Duck card START ________  -->
+        <div class="card">
+          <p class="name">Donald Duck</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">dduck</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">dduck</a>
```

Comment:
Insert your profile handle inside the `href` attribute. Just replace this `your_user_handle` with your user account handle.